src/esp/sim/Simulator.h

```diff
@@ -127,6 +127,22 @@ class Simulator {
     return metadataMediator_->getStageAttributesManager();
   }
 
+  /**
+   * @brief Get current active dataset name from @ref metadataMediator_.
+   */
+  std::string getActiveDatasetName() {
```

Comment:
Should these always be "SceneDataset" to reduce confusion?

Reply:
Exactly!